UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -293,6 +200,8 @@ void InitializeEnvironment()
 
             if (Communicator != null)
             {
+                Communicator.RegisterSideChannel(new EngineConfigurationChannel());
```

Comment:
Doesn't have to be in this PR, but we should think some more about how we expect users to register their own side channels. Might need a virtual call here that they can override.

Reply:
We could also go the route of having a side channel component (a little bit like what you did with the sensors)

Reply:
I like that idea, though if the academy won't be in the scene in the future, maybe that doesn't make sense.  I'm not sure though.